### PR TITLE
Update github.py

### PR DESCRIPTION
**Pull request by kdobolyi**
Hi JS,

Check our my regexaroni for pulling country names out of strings

### CODE REVIEW

**Comment by jspeed-meyers** [comment]
@anarkiwi, if you have time today, can you please review this PR?

File: github.py [hunk 1]
```diff
@@ -7,6 +7,9 @@
 import os
 
 import requests
+from country_list import countries_for_language
```
Comment (jspeed-meyers) [inline github.py:10]: @kdobolyi, can you please follow the PEP 8 guidelines for import order? https://www.python.org/dev/peps/pep-0008/#imports . Please let me know if you have questions; I'm glad to help. I'm sorry for not making it clear that this import ordering is preferred before you started work on this PR. Nice work on this feature and PR!

File: github.py [hunk 2]
```diff
@@ -39,6 +42,70 @@ def get_contributors(repo):
     return committers
 
 
+def get_country_from_location(location_string):
+    """Return country (Hungary; United States, etc) from a text containing a city and/or state and/or country.
+    Args:
+        user: a text containing a city and/or state and/or country
```
Comment (jspeed-meyers) [inline github.py:48]: This argument should be location_string, not user.

File: github.py [hunk 3]
```diff
@@ -39,6 +42,70 @@ def get_contributors(repo):
...
+        return pieces[-1].strip()
+    if pieces[-1].strip() in state_names:
+        return "United States"
+    if pieces[-1].strip() in state_abbrev:
+        return "United States"
+
+    return "NONE"
+
+
+    """query = urllib.parse.quote(",".join(pieces))
```
Comment (jspeed-meyers) [inline github.py:96]: Do you plan to leave this code in to use it later?

File: test_gitgeo.py [hunk 4]
```diff
@@ -73,6 +73,33 @@ def test_get_contributor_location(self):
...
+        assert location == "United States"
+
+        location = get_country_from_location("Atlanta, Georgia")
+        assert location == "United States"
+
+        location = get_country_from_location("Georgia")
+        assert location == "Georgia"
+
+    @pytest.mark.xfail  # test should fail, until functionality implemented
+    def test_get_contributor_country_nonstandard(self):
```
Comment (jspeed-meyers) [inline test_gitgeo.py:93]: Nice addition of all these extra tests, especially the non-standard tests.

**Review by jspeed-meyers — CHANGES_REQUESTED** [review]
Nice work! So I take it that you're still working on implementing functionality for cities and locations throughout the world? Do I have that right? Should I merge this PR in and a future PR can work on that functionality?

File: github.py (hunk 1 above)
Comment (kdobolyi) [reply github.py:10]: No, this is great -- I normally clean these up in a refactoring step weeks into the project. LMK if they're still in the wrong order (hopefully not).

File: github.py (hunk 3 above)
Comment (kdobolyi) [reply github.py:96]: you're right, I need to cut it out since I found a better option...this is a code smell :-)

**Pull request merged**